src/libs/websocket.ts

```diff
@@ -0,0 +1,14 @@
+import { Accessor, Context, createContext, Setter } from "solid-js";
+
+// FIXME: Don't pass message signal through the websocket context data.
```

Comment:
An alternative to this may be [stores](https://docs.solidjs.com/concepts/stores).

Reply:
I'll look into stores